checkpoint_1/Readme.md

```diff
@@ -1,50 +1,49 @@
 
 Create a directory named foo_bar
 
-    Please use markdown formatting listing the command(s) you ran to accomplish this
+   `mkdir foo-bar`
     
 Go into the foo_bar directory
-
-    Please use markdown formatting listing the command(s) you ran to accomplish this
+  
+  `cd foo-bar`
 
 For each question that is asked below, please provide the command you would run to find the answer, and then answer the question.
 
 1) What directory are you in?
-
-    Please use markdown formatting listing the command(s) you ran to accomplish this
+    `/Users/misikirmehari/workspace/davinci_coders_t2_2016/homework/learn_command_line_exercises/foo_bar`
 
 2) Are you in the class directory?
 
-    Please use markdown formatting listing the command(s) you ran to accomplish this
+   `yes`
```

Comment:
Also "I would run `pwd`, which shows me `...`, so I am not in the class directory, but I am in a subdirectory`